### PR TITLE
[serd] Update version to 0.32.2

### LINKED ISSUE #42529: [serd] update to v0.32.2
### Library name

serd

### New version number

0.32.2

### Other information that may be useful (release notes, etc...)

The VCPKG port is still at version 0.30.4 from Apr 26, 2020: https://github.com/drobilla/serd/blob/main/NEWS

### PR DESCRIPTION
**Pull request by FrankXie05**
Fix https://github.com/microsoft/vcpkg/issues/42529

Switch to the official `meson` build.
The usage has been tested successfully locally.

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] ~The "supports" clause reflects platforms that may be fixed by this new version.~
- [ ] ~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~
- [ ] ~Any patches that are no longer applied are deleted from the port's directory.~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/serd/usage [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The package serd can be imported via CMake FindPkgConfig module:
+
+    find_package(PkgConfig)
+    pkg_check_modules(serd REQUIRED IMPORTED_TARGET serd-0)
+    target_link_libraries(main PRIVATE PkgConfig::serd)
```
Comment (dg0yt) [inline ports/serd/usage:5]: I suggest to remove this file.
vcpkg tool's heuristics reliably detect pc files and prints information in an approved form which is different from this form.

File: ports/serd/portfile.cmake [hunk 2]
```diff
@@ -2,24 +2,21 @@ vcpkg_from_gitlab(
...
     SOURCE_PATH "${SOURCE_PATH}"
-    OPTIONS_DEBUG -DDISABLE_INSTALL_HEADERS=1
+    OPTIONS
+        -Ddocs=disabled
+        -Dtests=disabled
+        -Dhtml=disabled
 )
 
-vcpkg_cmake_install()
+vcpkg_install_meson()
```
Comment (dg0yt) [inline ports/serd/portfile.cmake:18]:
```suggestion
vcpkg_install_meson()
vpckg_fixup_pkgconfig()
```

**Review by LilyWangLL — APPROVED** [review]

**Comment by dg0yt** [comment]
This PR switches serd to meson, and updates vendored CMakeLists.txt for sord, sratom, lilv.

But sord, sratom and lilv are from the same vendor as serd, and upstream has meson, too. Shouldn't these ports be updated and switched at the same time?

In addition, removing the CMake config package which was offered until now might mean to break vcpkg users. Even when serd is used only as a transitive depedency, CMake users now need a pkg-config executable.

**Comment by FrankXie05** [comment]
This can be split into several PRs.

**Comment by FrankXie05** [comment]
Closed by https://github.com/microsoft/vcpkg/pull/42537

**Pull request closed without merging**